about.html

```diff
@@ -0,0 +1,205 @@
+            <li><a class="nav-links" href="#partners">Sponsor</a></li>
+            <li><a class="nav-links" href="#speakers">Guest</a></li>
+            <li><a class="nav-links join-namm-btn" href="index.html">Events254 Arena</a></li>
+          </ul>
+        </nav>
+      </header>
+
+  <main>
+    <section id="namm-2020-news" class="about">
+      <div class="about-contents">
+        <div class="supportive-contents">
+          <h2 class="header-1 ff-2 color-primary p-64">Events254 Arena in collaboration with KICC</h2>
+          <div class="about-desc-box bg-white">
+            <p class="about-desc color-dark-text">The Kenyatta International Convention Centre, formerly the Kenyatta International
+               Conference Centre, is a 28-story building located in Nairobi, Kenya. The KICC is located in the City Square of Nairobi,
+                and is a crucial address for a number of Government offices, including those of recently elected Senators
+                The Kenyatta International Convention Centre is an icon and landmark for Kenya. KICC boasts of being the leading 
+                facility in the meetings industry in East Africa and has held quite a number of regional and international conferences
+                 like the World Bank Conference in 1973, Inter-parliamentary Union summit in 2008, other UN based conferences, 8th 
+                 African Growth and Opportunity Act (AGOA), 7th Global Conference on health, the 5th Pan African Malaria conference, 
+                 Internet Corporation for Assigned Names and Numbers (ICANN) Congress & the 2010 International Federation of Biomedical
+                  Laboratory Scientists (IFBLS), 10th WTO Meeting, TICAD VI, 2nd High Level Meeting on GPEDC among others.
+              
+              </p>
+          </div>
+          <small class="small-content color-dark-text">contact us via Email for any further questions about Arena254.
+            </small>
+          <strong><u><a href="briannyachae168@gmail.com"><small
+                  class="color-secondary">briannyachae168@gmail.com</small></a></u></strong>
+        </div>
+      </div>
+    </section>
+
+    <section class="feature bg-white">
+      <article class="logo-info">
+        <div class="feature-logo">
+          <h3 class="color-secondary text-center">
+            The Kenyatta International Convention Centre (KICC)
+          </h3>
+          <div class="line bg-primary partners-line"></div>
+        </div>
+        <div class="logo-design">
+          <p class="text-center sm-fnt">The Kenyatta International Convention Centre (KICC)s was decided through the logo
+            competition
+            from 14th feb 1990 to 16th
+            may 1995.</p>
+        </div>
+        <div class="namm-logo ">
+          <img class="namm-logoo" src="./assets/images/logo.png" alt="logo">
+        </div>
+      </article>
+      <article class="pst-occurence">
```

Comment:
- [ ] Good job adding more cards 😉 , Kindly make alll cards have equal widths this will make your design have similar width to the Behance design 👍🏿 



Behance Design          |  Your design     
:-------------------------:|:-------------------------:
![Screenshot_26](https://user-images.githubusercontent.com/69638013/163430647-4335faed-4713-4429-81df-3eb8476aa645.png)  |  ![Screenshot_27](https://user-images.githubusercontent.com/69638013/163430673-f4bb2451-e751-48cd-97e0-95c4300cc271.png)

Reply:
I  have  fixed the problem thanks